DataViews/Program.cs

```diff
@@ -371,6 +375,74 @@ await foreach (var value in values)
                 }
 
                 #endregion //step11
+
+                // Step 12 - Add Units of Measure Column
+                #region step12
+                Console.WriteLine("Step 12: Add Units of Measure Column");
+                fieldSet = dataView.DataFieldSets.Single(a => a.QueryId == sampleQueryId);
```

Comment:
do we need a comment saying we first are finding the datafields we want to include uoms of?

Reply:
sure, i can add those